src/components/dashboard/Dashboard.js

```diff
@@ -782,16 +777,16 @@ const Dashboard = props => {
 const getStylesFromProps = ({ theme }) => ({
   headerWrapper: {
     height: '100%',
+    paddingBottom: getDesignRelativeHeight(
```

Comment:
no need for relative height
relative height should be used only where we want the whole design to fit inside smaller screens
in this case we have a scroll so its not an issue